docker/docker-compose.yml

```diff
@@ -15,7 +15,7 @@ services:
       http.cors.enabled: "true"
       http.cors.allow-credentials: "true"
       http.cors.allow-origin: "*"
-      http.cors.allow-methods: "GET"
+      http.cors.allow-methods: "GET, PUT"
```

Comment:
Do we really need this `PUT`?

Reply:
Given that this file is for development purposes only, yes.